src/domains/chains/components/ChainSelector.tsx

```diff
@@ -7,12 +7,13 @@ import { Definition } from 'src/domains/chains/utils/definitions';
 import supportedChains from 'src/domains/chains/utils/supportedChains';
 import useChain from 'src/domains/chains/utils/useChain';
 import Button from 'src/domains/misc/components/Button';
-import CIcon from 'src/domains/misc/components/CIcon';
 import SelectBox from 'src/domains/misc/components/SelectBox';
 import { BOTTOM_MENU_BREAKPOINT, BREAKPOINTS } from 'src/domains/misc/consts/consts';
 import { typography } from 'src/domains/styling/utils/tokens';
 import vars from 'src/domains/styling/utils/vars';
 
+import BIcon from '../../misc/components/BIcon';
```

Comment:
```suggestion
import BIcon from 'src/misc/components/BIcon';
```

here and everywhere else.